package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axelar-network/axelar-gmp-sdk-solidity",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
revert this version bump. the release PR will handle it

Reply:
got it. reverted version bump